install/update.php

```diff
@@ -2884,210 +3133,210 @@ function do_networkport_migration($migration) {
+                           "int(11) NOT NULL DEFAULT '0'");
+   $migration->addField($newTable,
+                           "ifmtu",
+                           "int(8) NOT NULL DEFAULT '0'");
+   $migration->addField($newTable,
+                           "ifspeed",
+                           "bigint(50) NOT NULL DEFAULT '0'");
+   $migration->addField($newTable,
+                           "ifinternalstatus",
+                           "varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL");
+   $migration->addField($newTable,
+                           "ifconnectionstatus",
+                           "int(8) NOT NULL DEFAULT '0'");
+   $migration->addField($newTable,
+                           "iflastchange",
+                           "varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL");
+   $migration->addField($newTable,
+                           "ifinoctets",
+                           "bigint(50) NOT NULL DEFAULT '0'");
+   $migration->addField($newTable,
+                           "ifinerrors",
+                           "bigint(50) NOT NULL DEFAULT '0'");
+   $migration->addField($newTable,
+                           "ifoutoctets",
+                           "bigint(50) NOT NULL DEFAULT '0'");
+   $migration->addField($newTable,
+                           "ifouterrors",
+                           "bigint(50) NOT NULL DEFAULT '0'");
+   $migration->addField($newTable,
+                           "ifstatus",
+                           "varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL");
+   $migration->addField($newTable,
+                           "mac",
+                           "varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL");
+   $migration->addField($newTable,
+                           "ifdescr",
+                           "varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL");
+   $migration->addField($newTable,
+                           "ifalias",
+                           "varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL");
+   $migration->addField($newTable,
+                           "portduplex",
+                           "varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL");
+   $migration->addField($newTable,
+                           "trunk",
+                           "tinyint(1) NOT NULL DEFAULT '0'");
+   $migration->addField($newTable,
+                           "lastup",
+                           "datetime DEFAULT NULL");
+   $migration->addKey($newTable,
+                        "networkports_id");
+   $migration->migrationOneTable($newTable);
+   $DB->list_fields($newTable, false);
+
+   /*
+    * Table glpi_plugin_fusioninventory_networkportlogs
+    */
+   $newTable = "glpi_plugin_fusioninventory_networkportlogs";
+   if ($DB->tableExists("glpi_plugin_tracker_snmp_history")) {
+      // **** Update history
+      update213to220_ConvertField($migration);
+
+      // **** Migration network history connections
+      $query = "SELECT count(ID) FROM `glpi_plugin_tracker_snmp_history`
```

Comment:
did you miss this query ?

Reply:
Yes... And no :p I've not replace all SELECT queries yet; espacially the more complicated. This one is not; and should be migrated.